packages/enhanced-img/src/vite-plugin.js

```diff
@@ -196,11 +185,12 @@ export function image_plugin(imagetools_plugin) {
 
 				return {
 					code: s.toString(),
-					map: s.generateMap()
+					map: s.generateMap({ hires: 'boundary', includeContent: false })
```

Comment:
what does `includeContent: false` do here? I looked at the magic string docs, but they don't mean anything to me

does `boundary` apply for markup? the magic string docs say "suitable for string semantics that are separated by words", but I'm not sure markup would be considered as "separated by words"

Reply:
I'm not quite sure what the impact of including or not including the original content is, but it looks to me like `includeContent` is `false` by default, so we can omit that. I'll do some testing on the `hires` piece
```suggestion
					map: s.generateMap({ hires: 'boundary' })
```

Reply:
the `hires: 'boundary'` piece looks fine to me. I couldn't tell any difference in error output between them